source/change-streams/tests/README.rst

```diff
@@ -164,43 +164,28 @@ The following tests have not yet been automated, but MUST still be tested
 11. - For a ``ChangeStream`` under these conditions:
       - Running against a server ``>=4.0.7``.
       - The batch is empty or has been iterated to the last document.
-   - Expected result: 
+    - Expected result:
```

Comment:
This was a formatting issue overlooked in #611. The Travis test wasn't smart enough to catch.